README.md
# fas2020

tcpdf/CHANGELOG.TXT
6.0.023 (2013-08-05)
	- GNU Freefont fonts were updated.
	- Licensing and copyright information about fonts were improved.
	- PNG image support was improved.

6.0.022 (2013-08-02)
	- fixing initialization problem for signature_appearance property.

6.0.021 (2013-07-18)
	- The bug caused by the preg_split function on some PHP 5.2.x versions was fixed.

6.0.020 (2013-06-04)
	- The method addTTFfont() was fixed (Bug item #813 Undefined offset).

6.0.019 (2013-06-04)
	- The magic constant __DIR__ was replaced with dirname(__FILE__) for php 5.2 compatibility.
	- The exceptions raised by file_exists() function were suppressed.

6.0.018 (2013-05-19)
	- The barcode examples were changed to automatically search for the barcode class path (in case the examples directory is not installed under the TCPDF fascalab_2020).

6.0.017 (2013-05-16)
	- The command line tool tcpdf_addfont.php was improved.
	- The php logic was removed from configuration files that now contains only constant defines.
	- The tcpdf_autoconfig.php file was added to automatically set missing configuration values.

6.0.016 (2013-05-15)
	- The tcpdf_addfont.php tool was improved (thanks to Remi Collet).
	- Constant K_PATH_IMAGES is now automatically set in configuration file.

6.0.015 (2013-05-14)
	- Some unused vars were removed from AddFont() method.
	- Some directories were moved inside the examples directory.
	- All examples were updated to reflect the new default structure.
	- Source code were clean-up up to be more compatible with system packaging.
	- Files encodings and permissions were reset.
	- The command line tool tcpdf_addfont.php was added on the tools directory.

6.0.014 (2013-04-13)
	- The signature of addTTFfont() method includes a new parameter to link existing fonts instead of copying and compressing them.

6.0.013 (2013-04-10)
	- Add support for SVG dx and dy text/tspan attributes.
	- replace require() with require_once().
	- fix some minor typos on documentation.
	- fix a problem when deleting all pages.

6.0.012 (2013-04-24)
	- An error condition in addHtmlLink() method was fixed (bug #799).

6.0.011 (2013-04-22)
	- Minor documentation changes.

6.0.010 (2013-04-03)
	- The method Rect() was fixed to print borders correctly.

6.0.009 (2013-04-02)
	- Adding back some files that were not properly committed on the latest release.

6.0.008 (2013-04-01)
	- Duplicated encoding maps was removed from tcpdf_font_data.php.
	- Fixing bug on AddTTFFont().

6.0.007 (2013-03-29)
	- HTML/CSS font size conversion were improved.

6.0.006 (2013-03-27)
	- Bug related to SVG and EPS files on xobjects were fixed.

6.0.005 (2013-03-26)
	- Default font path was fixed.

6.0.004 (2013-03-21)
	- Return value of addTTFfont() method was fixed.

6.0.003 (2013-03-20)
	- A bug related to non-unicode mode was fixed.

6.0.002 (2013-03-18)
	- _getFIXED call on tcpdf_fonts.php was fixed.

6.0.001 (2013-03-18)
	- Fixed $uni_type call on tcpdf.php.

6.0.000 (2013-03-17)
	- IMPORTANT: PHP4 support has been removed starting from this version.
	- Several TCPDF methods and vars were moved to new class files: tcpdf_static.php, tcpdf_colors.php, tcpdf_images.php, tcpdf_font_data.php, tcpdf_fonts.php.
	- Files htmlcolors.php, spotcolors.php, unicode_data.php and ecodings_maps.php were removed.
	- Barcode classes were renamed and new barcode examples were added.
	- Class TCPDF_PARSER was improved.

********************************************************************************

5.9.209 (2013-03-15)
	- Image method was improved.

5.9.208 (2013-03-15)
	- objclone fuction was patched to support old imagick extensions.
	- tcpdf_parser was improved to support Cross-Reference Streams and large streams.

5.9.207 (2013-03-04)
	- Datamatrix class was fixed (a debug echo was removed).

5.9.206 (2013-02-22)
	- Bug item #754 "PNG with alpha channel becomes gray scale" was fixed.
	- Minor documentation fixes.

5.9.205 (2013-02-06)
	- The constant K_TCPDF_THROW_EXCEPTION_ERROR was added on configuration file to change the behavior of Error() method.
	- PDF417 barcode bug was fixed.

5.9.204 (2013-01-23)
	- The method Bookmark() was extended to include named destinations, URLs, internal links or embedded files (see example n. 15).
	- automatic path calculation on configuration file was fixed.
	- Error() method was extended to throw new Exception if PHP > 5.

5.9.203 (2013-01-22)
	- Horizontal position of radiobuttons and checkboxes was adjusted.

5.9.202 (2012-12-16)
	- Vertical space problem after table was fixed.

5.9.201 (2012-12-10)
	- First 256 chars are now always included on font subset to overcome a problem reported on the forum.

5.9.200 (2012-12-05)
	- Bug item #768 "Rowspan with Pagebreak error" was fixed.
	- Page regions now works also with limited MultiCell() cells.

5.9.199 (2012-11-29)
	- Internal setImageBuffer() method was improved.

5.9.198 (2012-11-19)
	- Datamatrix EDIFACT mode was fixed.

5.9.197 (2012-11-06)
	- Bug item #756 "TCPDF 5.9.196 shows line on top of all PDFs" was fixed.

5.9.196 (2012-11-02)
	- Several methods were improved to avoid output when the context is out of page.
	- Bug item #755 "remove cached files before unsetting" was fixed.

5.9.195 (2012-10-24)
	- Method _putfonts() was improved.

5.9.194 (2012-10-23)
	- Text alignment on TextField() method was fixed.

5.9.193 (2012-09-25)
	- Support for named destinations on HTML links was added  (i.e.: <a href="#destinationname">link to named destination</a>).

5.9.192 (2012-09-24)
	- A problem on the releasing process was fixed.

5.9.191 (2012-09-24)
	- SVG image naow support svg and eps images.

5.9.190 (2012-09-23)
	- "page" word translation is now set to empty if not defined.
	- Tooltip feature was added on the radiobutton annotation.

5.9.189 (2012-09-18)
	- Bug item #3568969 "ini_get safe_mode error" was fixed.

5.9.188 (2012-09-15)
	- A datamatrix barcode bug was fixed.

5.9.187 (2012-09-14)
	- Subset feature was extended to include the first 256 characters.

5.9.186 (2012-09-13)

dv.php
<?php
ini_set('display_errors', 0);
error_reporting(0);
// require_once('functions.php'); 
session_start();  

$conn = mysqli_connect("localhost","fascalab_2020","w]zYV6X9{*BN","fascalab_2020");

$conn = mysqli_connect("localhost","fascalab_2020","7one@2019","fascalab_2020");

$username = $_SESSION['username'];
$select_user = mysqli_query($conn,"SELECT DIVISION_C FROM tblemployee WHERE UNAME = '$username'");
$rowdiv = mysqli_fetch_array($select_user);
$DIVISION_C = $rowdiv['DIVISION_C'];
if ($DIVISION_C == '10' || $DIVISION_C == '11' || $DIVISION_C == '12' || $DIVISION_C == '13' || $DIVISION_C == '14' || $DIVISION_C == '15' || $DIVISION_C == '16' ) {

  $pmo_id = 5;
}elseif ($DIVISION_C == '1' || $DIVISION_C == '2' || $DIVISION_C == '3' || $DIVISION_C == '5'  ) {

  $pmo_id = 1;
}elseif ($DIVISION_C == '18' ) {

  $pmo_id = 3;
}elseif ($DIVISION_C == '8' || $DIVISION_C == '17' ) {

  $pmo_id = 4;
}elseif ($DIVISION_C == '9' ) {

  $pmo_id = 6;
}elseif ($DIVISION_C == '7' ) {

  $pmo_id = 7;
}
if (isset($_POST['submit'])) {
  $po_no = $_POST['po_no'];
  $supplier = $_POST['supplier'];
  $purpose = $_POST['purpose'];
  $amount = $_POST['amount'];
  $address = $_POST['address'];

  $insert = mysqli_query($conn,"INSERT INTO dv(po_no,supplier,purpose,amount,address,office) VALUES('$po_no','$supplier','$purpose','$amount','$address','$pmo_id')");

  if ($insert) {
    echo ("<SCRIPT LANGUAGE='JavaScript'>
      window.alert('Successfuly Saved!')
      window.location.href = 'ViewDV.php';
      </SCRIPT>");

  }  else{
   echo ("<SCRIPT LANGUAGE='JavaScript'>
    window.alert('Error Occured in Saving');
    </SCRIPT>");
 }


}
?>

<!DOCTYPE html>
<html>
<body>
 <div class="box box-default">
  <div class="box-header with-border">
    <h1 align="">&nbspCreate DV</h1>
    <div class="box-header with-border">
    </div>
    <br>
    &nbsp &nbsp &nbsp   <li class="btn btn-success btn-s"><a href="ViewDV.php" style="color:white;text-decoration: none;">Back</a></li>

    <br>
    <br>
    <form method="POST" autocomplete="off" >
      <div class="box-body">
        <div class="well">
          <div class="row">
            <div class="col-md-6">
             <div class="form-group" hidden="">
              <label>Please Select : <small style="color:red;">*</small></label><br>
              <select name="burs_dv" class="form-control select2">
                <!-- <option></option> -->
                <option value="1">DV</option>
              </select>
            </div>
            <div class="form-group">
              <label>PO No. :  </label>
              <input autocomplete = "false"  class="form-control" name="po_no" type="text" id="po_no">

            </div>
            <div class="form-group">
              <label>Payee/Supplier : <small style="color:red;">*</small></label>
              <input required autocomplete = "false"  class="form-control" name="supplier" type="text" id="supplier">
            </div>

          </div>
          <div class="col-md-6">

           <div class="form-group">
            <label>Address :</label>
            <input required autocomplete = "false"  class="form-control" name="address" type="text" id="address">
          </div>

          <div class="form-group">
            <label>Particular/Purpose : <small style="color:red;">*</small></label>
            <input required autocomplete = "false"  class="form-control" name="purpose" type="text" id="purpose">
          </div>

          <div class="form-group">
            <label>Amount : <small style="color:red;">*</small></label>
            <input required autocomplete = "false"  class="form-control" name="amount" type="text" id="amount">
          </div>

        </div>


      </div>
    </div>
  </div>
  <button class="btn btn-primary btn-s" style="float: right;" id="finalizeButton" type="submit" name="submit" onclick="return confirm('Are you sure you want to save now?');">Save</button>
  <br>
</form>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>
<br>  
<br>
</body>

<!-- jQuery 3 -->
<script src="bower_components/jquery/dist/jquery.min.js"></script>
<!-- Bootstrap 3.3.7 -->
<script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
<!-- Select2 -->
<script src="bower_components/select2/dist/js/select2.full.min.js"></script>
<!-- InputMask -->
<script src="plugins/input-mask/jquery.inputmask.js"></script>
<script src="plugins/input-mask/jquery.inputmask.date.extensions.js"></script>
<script src="plugins/input-mask/jquery.inputmask.extensions.js"></script>
<!-- date-range-picker -->
<script src="bower_components/moment/min/moment.min.js"></script>
<script src="bower_components/bootstrap-daterangepicker/daterangepicker.js"></script>
<!-- bootstrap datepicker -->
<script src="bower_components/bootstrap-datepicker/dist/js/bootstrap-datepicker.min.js"></script>
<!-- bootstrap color picker -->
<script src="bower_components/bootstrap-colorpicker/dist/js/bootstrap-colorpicker.min.js"></script>
<!-- bootstrap time picker -->
<script src="plugins/timepicker/bootstrap-timepicker.min.js"></script>
<!-- SlimScroll -->
<script src="bower_components/jquery-slimscroll/jquery.slimscroll.min.js"></script>
<!-- iCheck 1.0.1 -->
<script src="plugins/iCheck/icheck.min.js"></script>
<!-- FastClick -->
<script src="bower_components/fastclick/lib/fastclick.js"></script>
<!-- AdminLTE App -->
<script src="dist/js/adminlte.min.js"></script>
<!-- AdminLTE for demo purposes -->
<script src="dist/js/demo.js"></script>
<!-- Page script -->
<script>
  $(function () {
    //Initialize Select2 Elements
    $('.select2').select2()

    //Datemask dd/mm/yyyy
    $('#datemask').inputmask('dd/mm/yyyy', { 'placeholder': 'mm/dd/yyyy' })
    //Datemask2 mm/dd/yyyy
    $('#datemask2').inputmask('mm/dd/yyyy', { 'placeholder': 'mm/dd/yyyy' })
    //Money Euro
    $('[data-mask]').inputmask()

    //Date range picker
    $('#reservation').daterangepicker()
    //Date range picker with time picker
    $('#reservationtime').daterangepicker({ timePicker: true, timePickerIncrement: 30, locale: { format: 'MM/DD/YYYY hh:mm A' }})
    //Date range as a button
    $('#daterange-btn').daterangepicker(
    {
      ranges   : {
        'Today'       : [moment(), moment()],
        'Yesterday'   : [moment().subtract(1, 'days'), moment().subtract(1, 'days')],
        'Last 7 Days' : [moment().subtract(6, 'days'), moment()],
        'Last 30 Days': [moment().subtract(29, 'days'), moment()],
        'This Month'  : [moment().startOf('month'), moment().endOf('month')],
        'Last Month'  : [moment().subtract(1, 'month').startOf('month'), moment().subtract(1, 'month').endOf('month')]
      },
      startDate: moment().subtract(29, 'days'),
      endDate  : moment()
    },
    function (start, end) {
      $('#daterange-btn span').html(start.format('MMMM D, YYYY') + ' - ' + end.format('MMMM D, YYYY'))
    }
    )

    //Date picker,
    $('#datepicker1').datepicker({
      autoclose: true
    })

    $('#datepicker2').datepicker({
      autoclose: true
    })
    $('#datepicker3').datepicker({
      autoclose: true
    })
    $('#datepicker4').datepicker({
      autoclose: true
    })

    //iCheck for checkbox and radio inputs
    $('input[type="checkbox"].minimal, input[type="radio"].minimal').iCheck({
      checkboxClass: 'icheckbox_minimal-blue',
      radioClass   : 'iradio_minimal-blue'
    })
    //Red color scheme for iCheck
    $('input[type="checkbox"].minimal-red, input[type="radio"].minimal-red').iCheck({
      checkboxClass: 'icheckbox_minimal-red',
      radioClass   : 'iradio_minimal-red'
    })
    //Flat red color scheme for iCheck
    $('input[type="checkbox"].flat-red, input[type="radio"].flat-red').iCheck({
      checkboxClass: 'icheckbox_flat-green',
      radioClass   : 'iradio_flat-green'
    })

    //Colorpicker
    $('.my-colorpicker1').colorpicker()
    //color picker with addon
    $('.my-colorpicker2').colorpicker()

    //Timepicker
    $('.timepicker').timepicker({
      showInputs: false
    })
  })
</script>
